components/api/src/backends/actix.rs

```diff
@@ -116,20 +116,11 @@ impl ExtendApiBackend for actix_web::Scope<()> {
 
 impl ResponseError for ApiError {
```

Comment:
I think we need some unit tests which check that `ApiError` is converted to the actix `HttpResponse` correctly.

The logic of this type is not as simple anymore.